fix(DB/Item) - Envolpe Assignment loot no longer gives you empty loot.

TheSCREWEDSoftware opened the pull request:
Now you won't get a empty bag but now you will get properly `You can't carry any more of those items.` and still create loot.

Now if you open any of the assignments envolpes and you already have that item, it will propely generate it and not give you an empty bag, see the images below to see the example and the table of items being changed

NOW: 
![image](https://github.com/user-attachments/assets/927cf0dd-d88f-40f0-824c-7dcd94581cac)

BEFORE:
![image](https://github.com/user-attachments/assets/e26a17ad-4184-49da-bea4-691813d0059c)

<details>
<summary>Click to me table of items changed</summary>

| Logistics                       | Combat                 | Tactical                    |
| ---------------------------- | ------------------------- | --------------------------- |
| Logistics Task Briefing I    | Combat Task Briefing I    | Tactical Task Briefing I    |
| Logistics Task Briefing I    | Combat Task Briefing I    | Tactical Task Briefing III  |
| Logistics Task Briefing II   | Combat Task Briefing II   | Tactical Task Briefing III  |
| Logistics Task Briefing II   | Combat Task Briefing II   | Tactical Task Briefing IV   |
| Logistics Task Briefing III  | Combat Task Briefing III  | Tactical Task Briefing IX   |
| Logistics Task Briefing III  | Combat Task Briefing IV   | Tactical Task Briefing V    |
| Logistics Task Briefing IV   | Combat Task Briefing IX   | Tactical Task Briefing VI   |
| Logistics Task Briefing IV   | Combat Task Briefing V    | Tactical Task Briefing VII  |
| Logistics Task Briefing IX   | Combat Task Briefing VI   | Tactical Task Briefing VIII |
| Logistics Task Briefing IX   | Combat Task Briefing VII  | Tactical Task Briefing X    |
| Logistics Task Briefing V    | Combat Task Briefing VIII |
| Logistics Task Briefing V    | Combat Task Briefing X    |                             |
| Logistics Task Briefing VI   | Combat Task Briefing XI   |                             |
| Logistics Task Briefing VI   | Combat Task Briefing XII  |                             |
| Logistics Task Briefing VII  |                           |                             |
| Logistics Task Briefing VII  |                           |                             |
| Logistics Task Briefing VIII |                           |                             |
| Logistics Task Briefing VIII |                           |                             |
| Logistics Task Briefing X    |                           |                             |
| Logistics Task Briefing X    |                           |                             |
| Logistics Task Briefing XI   |                           |                             |


</details>

## Changes Proposed:
<!-- If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address. -->
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:
<!-- If your fix has a relating issue, link it below -->
- Closes https://github.com/chromiecraft/chromiecraft/issues/7559
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/20758

## SOURCE:
<!-- If you can, include a source that can strengthen your claim -->
The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:
<!-- Does it build without errors? Did you test in-game? What did you test? On which OS did you test? Describe any other tests performed -->
This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.


## How to Test the Changes:
<!-- Describe in a detailed step-by-step order how to test the changes -->

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

- Make a character
- I would recommend you, having a empty inventory and big bags.
- `.additem 23162` // Foror's Crate of Endless Resist Gear Storage (36 slots) (2-3 bags)

Change the `1` to whatever value you want, you will need to test multiple of them.
- `.additem 20805 1` // Followup Logistics Assignment
- `.additem 20808 1` // Combat Assignment
- `.additem 20809 1` // Tactical Assignment
- `.additem 21131 1` // Followup Combat Assignment
- `.additem 21132 1` // Logistics Assignment
- `.additem 21133 1` // Followup Tactical Assignment
- `.additem 21266 1` // Logistics Assignment
- `.additem 21386 1` // Followup Logistics Assignment




## Known Issues and TODO List:
<!-- Is there anything else left to do after this PR? -->

- [ ]
- [ ]

<!-- If you intend to contribute repeatedly to our project, it is a good idea to join our discord channel. We set ranks for our contributors and give them access to special resources or knowledge: https://discord.com/invite/GyFvXpk7)
     Do not remove the instructions below about testing, they will help users to test your PR -->
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

Mervin31 commented:
> * `.additem 20805 1` // Followup Logistics Assignment
> * `.additem 20808 1` // Combat Assignment
> * `.additem 20809 1` // Tactical Assignment
> * `.additem 21131 1` // Followup Combat Assignment
> * `.additem 21132 1` // Logistics Assignment
> * `.additem 21133 1` // Followup Tactical Assignment
> * `.additem 21266 1` // Logistics Assignment
> * `.additem 21386 1` // Followup Logistics Assignment

why are there multiple ids to the same single item?
I.e.: 
Followup Logistics Assignment: 21386,20805
Logistics Assignment: 21132,21266

TheSCREWEDSoftware commented:
> why are there multiple ids to the same single item? I.e.: Followup Logistics Assignment: 21386,20805 Logistics Assignment: 21132,21266

Because some of the entries have the same name.

I'm trying this and i'm not sure if it's the correct query but logically it is (not at a pc with ac installed).

```sql
SELECT * FROM `item_template` WHERE `name` LIKE '%Briefing%'
```

But liek with hte wit craft PR, doesn't make sense to not updae all `Briefings` related to the Assignment envolopes. Even if they are un-used.

There's a table on my description when i extracted the IDs i extracted names and place them in the table so you can see what items are being changed.

sudlud commented:
three of the entries you're updating do not even have a `startquest` set, so setting the ignore-quest-status flag feels incorrect for those.

```
20946
21246
21247
```

TheSCREWEDSoftware commented:
But like the comment above and same with the wit pr ones even if they are un used the changes make sense to change all of them (and if anyone ever uses the un used items they will work out of the box)
